src/actions.js

```diff
@@ -1,47 +1,66 @@
 import fs from 'fs'
 
-export const handleFileLoading = e => {
-  return dispatch => {
+const getSubtitlesFileType = file => {
+  const [ , fileExtension ] = file.name.split(".")
+  let fileType
+  switch(fileExtension) {
```

Comment:
Vão entrar outras extensões nesse switch ou serão sempre duas extenções?

Reply:
Eu pretendo inserir suporte a mais extensões conforme eu for vendo as mais usadas.